tracer/src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -91,6 +91,10 @@
     <DefineConstants>$(DefineConstants);ASYNCLOCAL;HASHTABLE;FEATURE_DEFAULT_INTERFACE;FEATURE_SPAN</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);REGEXGENERATOR</DefineConstants>
```

Comment:
If we _do_ add a .NET 7 target, will this cause any issues? 🤔

Reply:
No, it should only affect 7.0 source code, not the binaries.